.gitignore

```diff
@@ -3,3 +3,5 @@ coverage
 *.swp
 *.swo
 todo.txt
+package-lock.json
```

Comment:
lets just checkout the `.gitignore` from this PR because we are not making changes to it anymore here. we can address the package-lock.json issue separately